karma.conf.js

```diff
@@ -6,15 +6,9 @@ module.exports = function(config) {
       'node_modules/expect.js/index.js',
       'node_modules/angular/angular.js',
       'node_modules/angular-mocks/angular-mocks.js',
-      'node_modules/d3/d3.js',
-      '.tmp/src/utils/*.js',
-      '.tmp/src/factories/*.js',
-      '.tmp/src/LineChart.js',
-      '.tmp/src/app.js',
-      '.tmp/test/unit/**/*.js',
-      {pattern: '.tmp/**/*.map', included: false},
```

Comment:
we should leave these 3 lines, because they are serving the *.map files. otherwise chrome is complaining about them